features/continuous-integration.md

```diff
@@ -1,2 +1,135 @@
+```yaml
+options:
+  setExitCode: true
+  organization: acme-co
+
+images:
+  docker:
+  - nginx:1.18-alpine
+  - quay.io/acme-co/my-app:$CI_HASH
+
+manifests:
+  yaml:
+  - ./deploy/test.yaml
+  helm:
+  - name: prod
+    path: ./deploy/chart
+    values:
+      foo: bar
+      resources:
+        requests:
+          memory: 1Gi
+```
+
+Next, you'll need to add the `FAIRWINDS_TOKEN` environment variable to your CI environment. This
+token can be found in the CI instructions in Insights, or on your organization's settings page. Note that
+most CI platforms provide a way to specify secrets in your environment variables.
+
+Last, you'll need to execute the Insights CI script within your CI pipeline.
+You may want to download, inspect, and store a copy of the script in your repository.
+```
+curl https://insights.fairwinds.com/static/insights-ci.sh | bash
+```
+
+Your repository will show up in the Insights UI once that script has been successfully run.
+
+## fairwinds-insights.yaml
+Here's a full list of options available in fairwinds-insights.yaml:
+
+### Options
+These are high-level options for the Insights CI integration.
+* `options.organization` - the name of your organization in Insights
+* `options.setExitCode` - set a non-zero exit code if issues are found. Default `false`
+* `options.baseBranch` - the branch to compare results to. Default `master`
+* `options.newActionItemThreshold` - the maximum number of new action items that can appear for a passing test. Default `0`
+* `options.severityThreshold` - if _any_ action item hits this threshold, the test will fail. Default `danger`
+* `options.junitOutput` - if specified, Insights will print JUnit XML results here
+* `options.repositoryName` - the name of the repository. Must match the name in GitHub if using the GitHub integration. Defaults to the output of `git remote -v`
+* `options.tempFolder` - a temporary directory to store files in. Default `./_insightsTemp/`
+* `options.hostname` - the host of the Insights instance to send results to. Default `https://insights.fairwinds.com`
+
+### Images
+Specify any images you'd like Insights to scan for vulnerabilities. These images must be available
+locally in your CI environment, either through running `docker build` or `docker pull`.
+
+* `images.docker` - an array of image names and tags to scan
+
+Note that you can include environment variables in your image names and tags. This is helpful
+if you set your image tag based on things like the current Git branch, tag, or commit SHA.
+
+### Manifests
+Specify any YAML or Helm manifests you'd like Insights to scan for configuration issues.
+Helm files can be templated using a variables file, or by specifying variables directly
+in your fairwinds-insights.yaml
+
+* `manifests.yaml` - an array of directories or files containing Kubernetes YAML manifests to scan
+* `manifests.helm` - an array of helm charts to template and scan
+* `manifests.helm[0].name` - a name to give this templated chart
```

Comment:
The helm section I wonder if the [0] isn't as clear as it could be? I'd lean towards [x] or [].